0.5.0 (#123)
italia/eidas-it-wallet-docs: docs/en/pid-eaa-issuance.rst

Reviewer: I am working on the security checks that must be performed by the Wallet Instance and PID/(Q)EAA issuer during the issuance phase. I need some more days to complete it. Would you like to go for the release without them or as soon as they are complete, I will do a pull request for the current release?

Author: yes, we'll have it in 0.6.0

I'd like to have this in 0.5.0:
https://github.com/italia/eudi-wallet-it-docs/issues/101

Below the description of the most relevant operations involved in the (Q)EAA issuance:

>     1. **Obtaining the trusted (Q)EAA Issuer**: the Wallet Instance discovers the trusted (Q)EAA Issuer ufing the Federaiton API (e.g.: using the Subordinate Listing Endpoint of the Trust Anchor and its Intermediates), then inspects the metadata looking for the credential capabilities. 

Reviewer 2: ```suggestion
    1. **Obtaining the trusted (Q)EAA Issuer**: the Wallet Instance discovers the trusted (Q)EAA Issuer using the Federation API (e.g.: using the Subordinate Listing Endpoint of the Trust Anchor and its Intermediates), then inspects the metadata looking for the credential capabilities. 
```

.. note::

>     **PID Credential Schema and Status registration:** The PID/(Q)EAA Issuer MUST register all the issued credentials for their later revocation. 

Reviewer 2: ```suggestion
    **PID/(Q)EAA Credential Schema and Status registration:** The PID/(Q)EAA Issuer MUST register all the issued credentials for their later revocation. 
```

>     **PID Credential Schema and Status registration:** The PID/(Q)EAA Issuer MUST register all the issued credentials for their later revocation. 

Reviewer 2: ```suggestion
    **PID/(Q)EAA Credential Schema and Status registration:** The PID/(Q)EAA Issuer MUST register all the issued credentials for their later revocation. 
```

Reviewer 2: We might remove this and deal with it in a new section on credential lifecycle, @peppelinux  what do you think?

Author: @fmarino-ipzs to be addressed in a separate issue

    - [`DPoP-draft16 <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-dpop-16>`_. Section 6.1] and [:rfc:`7638`].

Credential endpoint
-------------------

> The Credential Endpoint issues a credential as approved by the End-User upon presentation of a valid Access Token representing this approval, as defined in `OPENID4VCI`_.

Reviewer 2: @peppelinux , should credential be a defined term?

Author: no, since it is defined in this section with a dedicated section

Credential Request
^^^^^^^^^^^^^^^^^^^

The Wallet Instance when requests the PID/(Q)EAA to the PID/(Q)EAA Credential endpoint, MUST use the following parameters in the entity-body of the HTTP POST request, using the `application/json` media type.

> The Credential endpoint MUST accept and validate the *DPoP proof* sent in the DPoP field of the HTTP Header based on the steps defined in Section 4.3 of [DPoP-draft16]. The *DPoP proof* in addition to the values that are defined in the Token Endpoint section MUST contain the following claim:

Reviewer 2: ```suggestion
The Credential Endpoint MUST accept and validate the *DPoP proof* sent in the DPoP field of the HTTP Header based on the steps defined in Section 4.3 of [DPoP-draft16]. The *DPoP proof* in addition to the values that are defined in the Token Endpoint section MUST contain the following claim:
```

    - **Description**
    - **Reference**
  * - **format**
    - Format of the Credential to be issued. This MUST be set to `vc+sd-jwt` when the credential type is SD-JWT.
    - `[OIDC4VCI. Draft 13] <https://openid.bitbucket.io/connect/openid-4-verifiable-credential-issuance-1_0.html>`_.
  * - **credential**

>     - Contains the issued PID/(Q)EAA. When the credential type is SD-JWT, it MUST be an SD-JWT JSON Object (see Section :ref:`PID/(Q)EAA Data Model <pid_eaa_data_model.rst>`).

Reviewer 2: ```suggestion
    - Contains the issued PID/(Q)EAA. When the credential type is SD-JWT, it MUST be an `SD-JWT`_ JSON Object (see Section :ref:`PID/(Q)EAA Data Model <pid_eaa_data_model.rst>`).
```
